src/amo/components/SearchResult.js

```diff
@@ -28,11 +28,12 @@ export class SearchResultBase extends React.Component {
             <div className="SearchResult-rating">
               <Rating rating={addon.ratings.average} readOnly size="small" />
             </div>
-            <h3 className="SearchResult-author">{addon.authors[0].name}</h3>
-            <h3 className="SearchResult-users">{i18n.sprintf(
-              i18n.ngettext('%(total)s user', '%(total)s users', averageDailyUsers),
-              { total: i18n.formatNumber(averageDailyUsers) },
-            )}
+            <h3 className="SearchResult-author-users">
+              <span className="SearchResult-author">{addon.authors[0].name}</span>
```

Comment:
Actually to include the `<span>` tags you'll need to add more, so the `i18n` strings look more like:

```js
'%(addonAuthorStartSpan)s%(addonAuthor)s%(addonAuthorEndSpan)s%(separator)s%(addonUsersStartSpan)s%(addonUsers)s user%(addonUsersEndSpan)s'
```

😒